bdikit/mapping_algorithms/value_mapping/algorithms.py

```diff
@@ -126,10 +126,10 @@ def match(
             response_message = completion.choices[0].message.content
             try:
                 response_dict = ast.literal_eval(response_message)
-                if response_dict["term"] in target_values_set:
-                    matches.append(
-                        (current_value, response_dict["term"], response_dict["score"])
-                    )
+                target_value = response_dict["term"]
+                score = round(float(response_dict["score"]), 1)
```

Comment:
I didn't understand: why do we need to round the score value?

Reply:
Won't rounding before the comparison introduce a bug (e.g.,`round(0.151, 1) >= 0.2` will return `True` even though 0.151 is less than 0.2)? Or am I missing something?

Reply:
Nice catch, fixed!